tests/backup/backup_test.go

```diff
@@ -6433,29 +6433,40 @@ var _ = Describe("{RestartBackupPodDuringBackupSharing}", func() {
 // This testcase verifies schedule backup creation with a single namespace.
 var _ = Describe("{ScheduleBackupCreationSingleNS}", func() {
 	var (
-		contexts           []*scheduler.Context
-		appContexts        []*scheduler.Context
-		backupLocationName string
-		backupLocationUID  string
-		cloudCredUID       string
-		bkpNamespaces      []string
-		cloudAccountName   string
-		backupName         string
-		schBackupName      string
-		schPolicyUid       string
-		restoreName        string
-		clusterStatus      api.ClusterInfo_StatusInfo_Status
+		contexts             []*scheduler.Context
+		appContexts          []*scheduler.Context
+		backupLocationName   string
+		backupLocationUID    string
+		cloudCredUID         string
+		bkpNamespaces        []string
+		cloudAccountName     string
+		backupName           string
+		schBackupName        string
+		schPolicyUid         string
+		restoreName          string
+		clusterStatus        api.ClusterInfo_StatusInfo_Status
+		testrailID           int
+		namespaceMapping     map[string]string
+		labelSelectors       map[string]string
+		cloudCredUIDMap      map[string]string
+		backupLocationMap    map[string]string
+		timeStamp            string
+		periodicPolicyName   string
+		providers            []string
+		backupScheduleNames  []string
+		scheduledBackupNames []string
 	)
-	var testrailID = 58014 // testrailID corresponds to: https://portworx.testrail.net/index.php?/cases/view/58014
-	namespaceMapping := make(map[string]string)
-	labelSelectors := make(map[string]string)
-	cloudCredUIDMap := make(map[string]string)
-	backupLocationMap := make(map[string]string)
-	bkpNamespaces = make([]string, 0)
-	timeStamp := strconv.Itoa(int(time.Now().Unix()))
-	periodicPolicyName := fmt.Sprintf("%s-%s", "periodic", timeStamp)
 
 	JustBeforeEach(func() {
+		testrailID = 58014 // testrailID corresponds to: https://portworx.testrail.net/index.php?/cases/view/58014
+		namespaceMapping = make(map[string]string)
+		labelSelectors = make(map[string]string)
+		cloudCredUIDMap = make(map[string]string)
+		backupLocationMap = make(map[string]string)
+		bkpNamespaces = make([]string, 0)
+		timeStamp = strconv.Itoa(int(time.Now().Unix()))
```

Comment:
Do not declare `timeStamp` here. Use `time.Now().Unix()` directly where ever it is needed.

Reply:
It is from older version. I removed it now.